core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -2155,17 +2184,21 @@ CacheConfigurationException storeConfiguredHasBothReadAndWriteOnly(String storeC
 //   @Message(value = "Another partition or topology changed for while completing partial completed transaction. Retrying later.", id = 630)
 //   void topologyChangedPartitionHandlingTxCompletion();
 
-   @Message(value = "Cross-Site state transfer mode cannot be null.", id = 633)
+   @Message(value = "Cross-site state transfer mode cannot be null.", id = 633)
+   @Description("The mode attribute for cross-site state transfer configuration must have a value of AUTO or MANUAL. Modify the cache configuration with a valid value for the mode attribute.")
    CacheConfigurationException invalidXSiteStateTransferMode();
 
-   @Message(value = "Cross-Site automatic state transfer is not compatible with SYNC backup strategy.", id = 634)
+   @Message(value = "Cross-site automatic state transfer is not compatible with SYNC backup strategy.", id = 634)
+   @Description("Automatic state transfer is not possible if the backup strategy for cross-site replication is synchronous. Modify the cache configuration and set the state transfer mode to MANUAL. Alternatively you can change the backup strategy to use asynchronous mode.")
    CacheConfigurationException autoXSiteStateTransferModeNotAvailableInSync();
 
    @LogMessage(level = WARN)
    @Message(value = "[%s] Failed to receive a response from any nodes. Automatic cross-site state transfer to site '%s' is not started.", id = 635)
+   @Description("Before it starts automatic cross-site state transfer operations, the coordinator node checks all local nodes to determine if state transfer is necessary. This error occurs when the coordinator node gets an exception from one or more local nodes. Check that nodes in the cluster are online and operating as expected.")
    void unableToStartXSiteAutStateTransfer(String cacheName, String targetSite, @Cause Throwable t);
 
    @Message(value = "State transfer timeout (%d) must be greater than or equal to the remote timeout (%d)", id = 636)
+   @Description("State transfer operations cannot have a lower timeout value than the remote timeout. Need more info.")
```

Comment:
not related to cross-site since this is for local cluster state transfer.
it mean the startup waits for duration smaller than the remote timeout (network)